app/views/videos/advanced_search.html.haml

```diff
@@ -0,0 +1,51 @@
+%section.advanced_search.container
+  = form_tag advanced_search_videos_path, class: 'form-horizontal', method: :get do
+    %header
+      %h1 Advanced Search
+    .form-group
+      .col-sm-6
+        = search_field_tag :query, params[:query], placeholder: "Search ideos by title, description", class: "form-control"
+    .form-group
+      .col-sm-6
+        .checkbox
+          = label_tag :reviews do
+            = check_box_tag :reviews, true, params[:reviews].present?
+            Include Reviews
+    .form-group
+      .row.average_rating_label Average Rating
+      = label_tag :rating_from, "From", class: "col-sm-1 control-label"
+      .col-sm-2
+        = select_tag :rating_from, options_for_select((10..50).map { |num| num/10.0 }, params[:rating_from]), prompt: "-", class: "form-control"
+      = label_tag :rating_to, "To", class: "col-sm-1 control-label"
+      .col-sm-2
+        = select_tag :rating_to, options_for_select((10..50).map { |num| num/10.0 }, params[:rating_to]), prompt: "-", class: "form-control"
```

Comment:
I would prefer to simplify this line and the line 18.

`(10..50).map { |num| num/10.0 }` moving this to a helper would help it a bit.